packages/integration-test/test/lit-protocol.test.ts

```diff
@@ -0,0 +1,230 @@
+import { EthereumPrivateKeySignatureProvider } from '@requestnetwork/epk-signature';
+import { LitProtocolCipherProvider } from '@requestnetwork/lit-protocol-cipher';
+import { RequestNetwork, Types, Utils } from '@requestnetwork/request-client.js';
+import { ethers } from 'ethers';
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+
+const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+async function waitForConfirmation(request: any, maxAttempts = 10, delayMs = 1000): Promise<void> {
+  let attempts = 0;
+  while (attempts < maxAttempts) {
+    try {
+      const data = await request.getData();
+      if (
+        data.state === Types.RequestLogic.STATE.CREATED ||
+        data.state === Types.RequestLogic.STATE.PENDING
```

Comment:
Normally, I wouldn't consider a request confirmed until it's CREATED. 

```suggestion
        data.state === Types.RequestLogic.STATE.CREATED
```